proto/sifnode/clp/v1/querier.proto

```diff
@@ -0,0 +1,60 @@
+syntax = "proto3";
+package sifnode.clp.v1;
+
+import "gogoproto/gogo.proto";
+import "sifnode/clp/v1/types.proto";
+
+option go_package = "github.com/sifchain/sifnode/x/clp/types";
+
+service Query {
+  rpc QueryGetPool(QueryGetPoolRequest) returns (QueryGetPoolResponse);
```

Comment:
I think the guideline is to leave off "query" in the roc name, unless this is for compatibility?